src/Actions/DuplicateEntry.php

```diff
@@ -21,7 +21,7 @@ public function visibleTo($item)
 
     public function confirmationText()
     {
-        $hasDescendants = $this->items
+        $hasDescendants = collect(config('statamic.sites'))->count() > 1 && $this->items
```

Comment:
```suggestion
        $hasDescendants = Site::hasMultiple() && $this->items
```

(and add `use Statamic\Facades\Site;`)

Reply:
I was looking for a method like this, but couldn't find it. Thank you :)